include/boost/gil/extension/numeric/affine.hpp

```diff
@@ -108,6 +108,36 @@ boost::gil::matrix3x2<T> inverse(boost::gil::matrix3x2<T> m)
     return res;
 }
 
+//feature
+//rotate an image from its center point for -180 deg <= theta <= 180 deg
+//using consecutive affine transformations
+//As top-left corner of source image is taken as origin, idea is to rotate the source image about the origin and then translate the new image
+//according to the mentioned rotation angle, and to re-center and fit the image in the source dimensions 
+// preferably use type double for less precision errors
```

Comment:
- `//<space>`
- Please, check the guidelines about the line length limit

Reply:
I have rewritten all of the comments.

Reply:
By the way, no need to comment about what you have done, just hit the "Resolve conversation" and the updated diff will show what's been done exactly.

Thanks